ref/general/packaging-deploying.adoc

```diff
@@ -36,24 +34,24 @@ Then, package your application with Maven or Gradle. The result is a WAR deploym
 
 [#config-ol-war]
 == Configuring Open Liberty to run a WAR file
-If you don't plan on containerizing your application for the cloud, you can configure Open Liberty to run the application from a WAR file.
-Before you can run an application from a WAR file on Open Liberty, you must change the default HTTP port of the server to the port where the application will be accessed by replacing the `httpPort` attribute value in the `server.xml` file.
-The following example updates the HTTP port of the server to `9090`:
+To deploy and run your application from a WAR file, first you must configure Open Liberty by defining the application with the `webApplication` element in the `server.xml` file.
+The following example configures the `helloworld` application:
 
 [source,xml]
 ----
-<httpEndpoint id="defaultHttpEndpoint"
-  host="localhost"
-  httpPort="9090"
-  httpsPort="9443" />
+<webApplication contextRoot="helloworld" location="helloworld.war" />
 ----
 
-Next, define the application with the `webApplication` element in the `server.xml` file.
-The following example configures the `helloworld` application:
+You can change the default HTTP port of the server to the port where the application will be accessed by replacing the `httpPort` attribute value in the `server.xml` file.
```

Comment:
`where the application will be accessed by...`

edit to remove future tense

`where the application is accessed by...`